tests/Domain/NoteTest.php

```diff
@@ -0,0 +1,307 @@
+use Notes\Collection\Collection as Collection;
+
+class NoteTest extends \PHPUnit_Extensions_Database_TestCase
+{
+    private $connection;
+    
+    public function getConnection()
+    {
+        $config     = new Configuration("app/Config/config.json");
+        $configData = $config->get();
+        $dbHost     = $configData['dbHost'];
+        $dbName     = $configData['dbName'];
+        $hostString = "mysql:host=$dbHost;dbname=$dbName";
+        
+        try {
+            $this->connection = new \PDO($hostString, $configData['dbUser'], $configData['dbPassword']);
+            $this->connection->setAttribute(\PDO::ATTR_ERRMODE, \PDO::ERRMODE_EXCEPTION);
+            $this->connection->exec("set foreign_key_checks=0");
+            return $this->createDefaultDBConnection($this->connection, $dbName);
+        } catch (\PDOException $e) {
+            echo "Connection failed: " . $e->getMessage();
+        }
+    }
+    
+    public function getDataSet()
+    {
+        return $this->createXMLDataSet(dirname(__FILE__) . '/_files/domain_note_seed.xml');
+    }
+    
+    public function testCanCreate()
+    {
+        $noteInput = array(
+            'userId' => 1,
+            'title' => 'Exception',
+            'body' => 'Creating a custom exception handler is quite simple.'
+        );
+        
+        $tagsInput    = array(
+            'PHP',
+            'PHP6'
+        );
+        $UserTagModel = array();
+        
+        $noteModel = new NoteModel();
+        $noteModel->setUserId($noteInput['userId']);
+        $noteModel->setTitle($noteInput['title']);
+        $noteModel->setBody($noteInput['body']);
+        
+        $noteDomain      = new NoteDomain();
+        $actualResultSet = $noteDomain->create($noteModel, $tagsInput);
+        
+        $noteModel = $actualResultSet[0];
+        
+        $userTagModel  = $actualResultSet[1];
+        $userTagModel1 = $userTagModel[0];
+        $userTagModel2 = $userTagModel[1];
+        
+        $noteTagModel  = $actualResultSet[2];
+        $noteTagModel1 = $userTagModel[0];
+        $noteTagModel2 = $userTagModel[1];
+        
+        $collection = new Collection();
+        $collection->add($noteModel);
+        $this->assertEquals($noteModel, $collection->getRow(0));
+        
+        $collection = new Collection();
+        $collection->add($userTagModel1);
+        $this->assertEquals($userTagModel1, $collection->getRow(0));
+        $collection = new Collection();
+        $collection->add($userTagModel2);
+        $this->assertEquals($userTagModel2, $collection->getRow(0));
+        
+        $collection = new Collection();
+        $collection->add($noteTagModel1);
+        $this->assertEquals($noteTagModel1, $collection->getRow(0));
+        
+        $collection = new Collection();
+        $collection->add($noteTagModel2);
+        $this->assertEquals($noteTagModel2, $collection->getRow(0));
+    }
+    
+    public function testCanCreatebyPassingEmptyArrayTag()
```

Comment:
change the name of the test case.Name of the test case could be,it_should_create_note_by_passing_an_empty_array_tag. remove the test name and add the test word in to comments.